actors/miner/tests/deadline_cron.rs

```diff
@@ -39,6 +41,71 @@ fn cron_on_inactive_state() {
     h.check_state(&rt);
 }
 
+#[test]
+fn test_vesting_on_cron() {
+    let mut h = ActorHarness::new(PERIOD_OFFSET);
+    let rt = h.new_runtime();
+    rt.set_balance(BIG_BALANCE.clone());
+    h.construct_and_verify(&rt);
+
+    // onboard some sectors and get power
+    let active_sectors =
+        h.commit_and_prove_sectors(&rt, 2, DEFAULT_SECTOR_EXPIRATION as u64, vec![], true);
+    // advance cron to activate power.
+    h.advance_and_submit_posts(&rt, &active_sectors);
+
+    // --- ADD REWARDS FOR VESTING
+    let apply_reward_fn = |target_epoch: ChainEpoch| {
+        rt.set_epoch(target_epoch);
+        h.apply_rewards(&rt, BIG_REWARDS.clone(), TokenAmount::zero());
+    };
+
+    let current_epoch = *rt.epoch.borrow();
+    apply_reward_fn(current_epoch);
+
+    // MOVE EPOCH BY HALF A DAY AND ADD ONE MORE REWARD SCHEDULE
+    let current_epoch = current_epoch + (EPOCHS_IN_DAY / 2) + 100;
+    apply_reward_fn(current_epoch);
+
+    // --- ASSERT FUNDS TO BE VESTED
+    let st = h.get_state(&rt);
+    let vesting_funds = st.load_vesting_funds(&rt.store).unwrap();
+    assert_eq!(360, vesting_funds.funds.len());
+
+    let q = QuantSpec { unit: REWARD_VESTING_SPEC.quantization, offset: st.proving_period_start };
+
+    let assert_locked_fn = |new_epoch: ChainEpoch, should_vest: bool| {
+        let st = h.get_state(&rt);
+        rt.set_epoch(new_epoch);
+        let new_deadline_info = st.deadline_info(rt.policy(), new_epoch + 1);
+        let old_funds = st.locked_funds;
```

Comment:
Nit: rename to `old_locked` and `new_locked`.

Reply:
Done.